app/assets/stylesheets/components/ma-says/_ma-says.scss

```diff
@@ -8,6 +8,7 @@
   background: asset-url('ma-says.png') 0 0 no-repeat;
   background-image: linear-gradient(transparent, transparent), asset-url('ma-says.svg');
   background-repeat: no-repeat;
+  background-size: 193px 75px;
```

Comment:
This is needed too, as i've realised that it needs to be re-sized down on the homepage where there is a ma says link in the campaign section.